client/components/App.jsx

```diff
@@ -10,9 +10,8 @@ const AppStyledForPricePromo = styled.div`
   width:370.5px;
   height: 431.5px;
   background: #1b1e1b;
-  margin-top: 20px;
 `;
-//padding-top:20px;  -- per chat with Micko pad such that the entire module floats down 20 pixels to align with his video monitor on my left
```

Comment:
it looks like you've removed margin/padding for top, I think in this case it will not align with my viewer, unless you have the margin and padding somewhere else then this is all good :)

Reply:
Yah, tried and tried to align it, but it's not working. For now, pushing and will try to figure out why it's not shifting into place.

Reply:
Okay, totally up to you :)